dom/react.js

```diff
@@ -16,10 +17,25 @@ const nativeTextareaValueSetter = Object.getOwnPropertyDescriptor(HTMLTextAreaEl
 
 /**
  * Sets the value of a textarea input element which has been manipulated by React.
+ * Deprecated: Use setReactInputSelectTextareaValue instead
  * @param {HTMLTextAreaElement} input 
  * @param {string} value 
  */
 export function setReactTextareaValue(input, value) {
 	nativeTextareaValueSetter.call(input, value);
 	input.dispatchEvent(new Event('input', { bubbles: true }));
 }
+
+// MB React input logics often requires: input event, update value, then change event
+// https://github.com/jesus2099/konami-command/blob/a94c502bd7d5f4f4e29c91c7ac2a52881c96821e/lib/SUPER.js#L163-L171
+
+/**
+ * Sets the value of an input/select/textarea element which has been manipulated by React.
+ * @param {HTMLInputElement|HTMLSelectElement|HTMLTextAreaElement} input
+ * @param {string} value
+ */
+export function setReactInputSelectTextareaValue(input, value) {
```

Comment:
I think we can include the explanation in the function's documentation comment and give it a shorter name to make it clear that this one is specifically for some MBS React inputs, something like:
```suggestion
/**
 * Sets the value of a MusicBrainz input/select/textarea element which has been manipulated by React.
 *
 * MB React input logics often requires: input event, update value, then change event
 *
 * @param {HTMLInputElement|HTMLSelectElement|HTMLTextAreaElement} input
 * @param {string} value
 * @see https://github.com/jesus2099/konami-command/blob/a94c502bd7d5f4f4e29c91c7ac2a52881c96821e/lib/SUPER.js#L163-L171
 */
export function setMBReactInputValue(input, value) {
```